lib/tasks/unlock_invite_statuses.rake

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+# This task is run inside Heroku Scheduler at regular intervals.
+# See Heroku Scheduler docs at: https://devcenter.heroku.com/articles/scheduler
+
+# Unlocks `InviteStatus` and `GroupInviteStatus` records that have been locked (stuck) for more than an hour.
+# This task sets locked statuses to the "unaccepted" status.
+task unlock_invite_statuses: :environment do
```

Comment:
I'd suggest moving this logic out to either the model or a service object so you can more easily write tests for it then call that code from your rake task.

Reply:
I suppose this would make more sense as a "service", although I am not sure what that entails yet. I'll have to read up.